scripts/image

```diff
@@ -145,28 +145,32 @@ fi
 echo "${TARGET_VERSION}" > ${INSTALL}/etc/release
 
 # Create /etc/os-release
-echo -e "NAME=\"${DISTRONAME}\"" > ${INSTALL}/etc/os-release
-echo -e "VERSION=\"${LIBREELEC_VERSION}\"" >> ${INSTALL}/etc/os-release
-echo -e "ID=\"libreelec\"" >> ${INSTALL}/etc/os-release
-echo -e "VERSION_ID=\"${OS_VERSION}\"" >> ${INSTALL}/etc/os-release
-echo -e "PRETTY_NAME=\"${DISTRONAME} (${LIBREELEC_BUILD}): ${LIBREELEC_VERSION}\"" >> ${INSTALL}/etc/os-release
-echo -e "HOME_URL=\"https://libreelec.tv\"" >> ${INSTALL}/etc/os-release
-echo -e "BUG_REPORT_URL=\"${ORIGIN_URL}\"" >> ${INSTALL}/etc/os-release
-echo -e "BUILD_ID=\"${GIT_HASH}\"" >> ${INSTALL}/etc/os-release
-echo -e "LIBREELEC_ARCH=\"${LIBREELEC_ARCH}\"" >> ${INSTALL}/etc/os-release
-echo -e "LIBREELEC_BUILD=\"${LIBREELEC_BUILD}\"" >> ${INSTALL}/etc/os-release
-echo -e "LIBREELEC_PROJECT=\"${PROJECT}\"" >> ${INSTALL}/etc/os-release
+cat <<EOF >${INSTALL}/etc/os-release
+NAME="${DISTRONAME}"
+VERSION="${LIBREELEC_VERSION}"
+ID="libreelec"
+VERSION_ID="${OS_VERSION}"
+PRETTY_NAME="${DISTRONAME} (${LIBREELEC_BUILD}): ${LIBREELEC_VERSION}"
+HOME_URL="https://libreelec.tv"
+BUG_REPORT_URL="${ORIGIN_URL}"
+BUILD_ID="${GIT_HASH}"
+LIBREELEC_ARCH="${LIBREELEC_ARCH}"
+LIBREELEC_BUILD="${LIBREELEC_BUILD}"
+LIBREELEC_PROJECT="${PROJECT}"
+EOF
```

Comment:
Command grouping instead?
```
{
echo ...
echo ...
echo ...
} >> /etc/os-release
```

This can encompass the [ -n ...... ] statements too to make one write to file.

Reply:
With command grouping, the `echo` would still need to escape the double quotes, which makes the resulting code rather ugly (and more prone to error), but yes the command grouping would eliminate the repeated redirection. Unfortunately neither solution is perfect, but the HERE doc does at least eliminate the vast majority of the escape and redirection issues, leaving just the 3 conditionals as the exception.

I'll drop the `-e` option from the 3x echo statements, though, as that option is completely unnecessary.

Reply:
`-e` dropped.